archiver/main.go

```diff
@@ -497,9 +559,9 @@ func flattenRecipient(recipient KnowBe4Recipient) (KnowBe4FlatRecipient, error)
 	return flatRecipient, nil
 }
 
-func flattenCampaigns(recipients []KnowBe4Campaign) ([]KnowBe4FlatCampaign, error) {
-	flatCampaigns := make([]KnowBe4FlatCampaign, len(recipients))
-	for i, recipient := range recipients {
+func flattenCampaigns(campaigns []KnowBe4Campaign) ([]KnowBe4FlatCampaign, error) {
+	flatCampaigns := make([]KnowBe4FlatCampaign, len(campaigns))
+	for i, recipient := range campaigns {
```

Comment:
should `recipient` here be `campaign`  or just `c`?

Reply:
Yep